src/nemo_evaluator/orchestration/orchestrator.py

```
@@ -743,12 +743,19 @@ def _resolve_service_connection(
     return url, config.get_model_id(service_name), config.get_api_key(service_name), config.get_service(service_name)
 
 
+def _adapter_proxy_listen_host(config: EvalConfig | None) -> str:
+    if config is not None and any(getattr(bench.sandbox, "type", None) == "docker" for bench in config.benchmarks):
+        return "0.0.0.0"
```

Ruff (0.15.20)

[error] Possible binding to all interfaces

(S104)